tests/TestCase/TestSuite/IntegrationTestCaseTest.php

```diff
@@ -178,6 +178,7 @@ public function testAssertRedirect()
         $this->_response = new Response();
         $this->_response->header('Location', 'http://localhost/tasks/index');
 
+        $this->assertRedirect();
```

Comment:
Wouldn't the assertNoRedirect() then has to be `assertHasNoRedirect()` to be the opposite equivalent - and having a consistent naming scheme?